CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I think this is a really good thing to have! Thoughts on adding a line on what happens if a community leader violates the code?

Reply:
That is spelled out in the link document, but sure, might be useful to have a sentence here too. Thanks

Reply:
Oh I misread your comment

Reply:
What _should_ happen? I've been looking through a [variety](https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php) of [online](https://ropensci.org/code-of-conduct/) [codes](https://www.hiretechladies.com/code-of-conduct/) of [conduct](https://www.jenkins.io/project/conduct/) and not seeing this addressed anywhere.

Reply:
It's obvious, I think, that the code of conduct applies equally to leadership as it does to other contributors.

Reply:
True, my thought was that they wouldn't be part of a decision on consequences (which is obvious, I think) but also if there is a violation, do they lose certain leadership privileges? Depending on the severity of action, of course.

Reply:
@leeyap See text change I just pushed. Feedback welcome.

Reply:
Looks good, thanks!